plugins/codeclimate/README.md

```diff
@@ -0,0 +1,103 @@
+# Codeclimate Plugin
+
+The Codeclimate Plugin displays a few stats from the quality section from [Codeclimate](https://codeclimate.com).
```

Comment:
```suggestion
The Code Climate Plugin displays a few stats from the quality section from [Code Climate](https://codeclimate.com).
```

Reply:
Good catch, yeah I should have payed more attention to how they presented the name `Code Climate`, should be resolved now.